instrumentation/opentelemetry-instrumentation-botocore/tests/test_botocore_instrumentation.py

```diff
@@ -150,8 +150,8 @@ def test_s3_client(self):
         spans = self.memory_exporter.get_finished_spans()
         assert spans
         span = spans[2]
-        self.assertEqual(
-            span.attributes,
```

Comment:
these are failing now as the underlying library is generating a unique ID per request which is recorded as an attribute so we can no longer use `assertEqual`.